### ISSUE TITLE
npm audit warning from js-yaml

### ISSUE THREAD

**Issue by rtm** [salesforce/sfdx-lwc-jest#131]
```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Denial of Service                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ js-yaml                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=3.13.0                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ @salesforce/sfdx-lwc-jest [dev]                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ @salesforce/sfdx-lwc-jest > @lwc/compiler >                  │
│               │ @lwc/style-compiler > cssnano > postcss-svgo > svgo >        │
│               │ js-yaml                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/788                             │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Code Injection                                               │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ js-yaml                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=3.13.1                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ @salesforce/sfdx-lwc-jest [dev]                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ @salesforce/sfdx-lwc-jest > @lwc/compiler >                  │
│               │ @lwc/style-compiler > cssnano > postcss-svgo > svgo >        │
│               │ js-yaml                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/813                             │
```

**Comment by trevor-bliss** [comment]
Are you seeing this on master? The version of `js-yaml` is 3.13.1 according to the yarn.lock which should be fine for both of these alerts.

https://github.com/salesforce/sfdx-lwc-jest/blob/master/yarn.lock#L3190-L3191

**Comment by pmdartus** [comment]
`js-yaml` is not an issue in this repository because of the `package.json` forcing resolution of the library to the `3.13.1` version: 

https://github.com/salesforce/sfdx-lwc-jest/blob/8d697580a4d129724d18be45e710af16cdb6cbc2/package.json#L44

The `cssnano` dependency that is pulling down `js-yaml` has been updated with LWC `1.1.6`. This issue will be automatically fixed next release as this project will get the new version of LWC.

**Comment by trevor-bliss** [comment]
This will be fixed once the prerelease branch is merged into master (after R2 in July)

**Comment by trevor-bliss** [comment]
0 vulnerabilities on latest master when running `npm audit`
